spec/pc-spec.js

```diff
@@ -3,12 +3,58 @@ var PC = rewire('../lib/pc');
 var Promise = require('bluebird');
 
 describe('Paper Cassette', function () {
+  var mockFiles = {
+    'some/directory/cactus.xml': {
+      contents: '<xml>Cactus</xml>',
+    },
+    'some/diva/falcon.xml': {
+      contents: '<xml>Falcon</xml>',
+    },
+    'some/diva/cbl.xml': {
+      contents: '<xml>CBL</xml>',
+    },
+    'some/buzz.xml': {
+      contents: '<xml>buzz</xml>',
+    },
+    'some/diva/directory/blamaste.xml': {
+      contents: '<xml>Blamaste</xml>',
+    },
+    'some/diva/president/oblama.xml': {
+      contents: '<xml>oblama</xml>',
+    },
+    'some/bad-blammajamma.xml': {
+      contents: '<xml>bad-blamma-jamma</xml>',
+    },
```

Comment:
For completeness:
```
{
    'some/blt.xml': {
      contents: '<xml>blt</xml>',
    },
}
```

An aside: would this be better inside a test fixture? :zap: